scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1604,5 +1645,11 @@ def addDBVersion():
     cur_audit = audit_conn.cursor()
     cur_visibility = visibility_conn.cursor()
     spindb = get_sql_db_conn()
-    perform_migration()
+
+   #check if it is pre-upgrade DB Update or post-upgrade Data Migration
+    if migrate_data_flag == 'true':        
```

Comment:
@sudhakaropsmx this has to be other way...false for update_db and true forperform_migration

Reply:
@rakeshkrmeena  only this way is working